R-package/src/cmake/modules/FindLibR.cmake

```diff
@@ -48,14 +49,16 @@ function(create_rlib_for_msvc)
       \nDo you have Rtools installed with its MinGW's bin/ in PATH?")
   endif()
 
+  set(R_MSVC_CORE_LIBRARY "${CMAKE_CURRENT_BINARY_DIR}/R.lib" CACHE PATH "R.lib file location")
```

Comment:
I'm sorry I was mistaken in my previous comment about the name. It should follow naming style from the official dev guide and start with `LIBR_` (not `R_`) like all other output variables.
https://cmake.org/cmake/help/v3.0/manual/cmake-developer.7.html#find-modules

Also, I find `location` a little bit misleading in terms that it refers to "directory", "place", IMHO.

All other changes look OK to me! Thanks!

Reply:
ok how about https://github.com/microsoft/LightGBM/pull/2963/commits/8936a8a3e8ffba3e1e3b8c1c168d703dcbcc39a4